### PR TITLE
[⚠️ Needs testing]Feature/biotope element

### LINKED ISSUE #76: Add biotope-element
## Expected behavior
I want to clone biotope and be ready to use biotope-element

## Actual behavior
I have to update a bunch of stuff to use biotope-element

### PR DESCRIPTION
**Pull request by SheepFromHeaven**
After cloning this version, one is able to use biotope element.
resolves #76

### CODE REVIEW

File: src/layouts/includes/htmlhead.scripts.hbs [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 <!-- build:js(.tmp) resources/js/scripts.head.all.min.js  -->
```
Comment (jurekbarth) [inline src/layouts/includes/htmlhead.scripts.hbs]: We need polyfills for IE11 for promises, fetch and Object.assign

```
<script nomodule src="resources/js/polyfills/object-assign.polyfill.js"></script>
<script nomodule src="https://cdn.jsdelivr.net/npm/whatwg-fetch@3.0.0/dist/fetch.umd.min.js"></script>
<script nomodule src="https://cdn.jsdelivr.net/npm/promise-polyfill@8/dist/polyfill.min.js"></script>
```

File: tsconfig.json [hunk 2]
```diff
@@ -1,29 +1,22 @@
 {
 	"compilerOptions": {
 		"allowJs": true,
-		"jsx": "React",
-		"module": "CommonJS",
+		"jsx": "react",
+		"module": "es6",
```
Comment (jurekbarth) [inline tsconfig.json]: I think this will break IE11, but i'm unsure 🙃

File: tsconfig.json [hunk 3]
```diff
@@ -1,29 +1,22 @@
...
 		"node_modules",
 		"patterns",
 		"test",
 		"dist",
 		".tmp",
 		"**/*.spec.ts"
+	],
+	"typeRoots": [
+		"node_modules/@types",
+		"typings"
```
Comment (jurekbarth) [inline tsconfig.json]: Does that work, with `types/global.d.ts`🙃

File: src/layouts/includes/htmlhead.scripts.hbs (hunk 1 above)
Comment (SheepFromHeaven) [reply src/layouts/includes/htmlhead.scripts.hbs]: @jurekbarth shouldn't we also fetch the object assign polyfill from a cdn?

Comment (jurekbarth) [reply src/layouts/includes/htmlhead.scripts.hbs]: @SheepFromHeaven i'm fine with both versions, this was just an example. Depending on the use-case both versions have its pros and cons. Sorry for the confusion using both possibilities 🙃We should start docs to document the polyfills folder btw.

File: src/layouts/includes/htmlhead.scripts.hbs [hunk 4]
```diff
@@ -2,6 +2,10 @@
...
 <script src="resources/js/vendor/modernizr.js"></script>
+<script src="https://unpkg.com/@webcomponents/webcomponentsjs@2.2.1/webcomponents-loader.js"></script>
+<script nomodule src="https://cdn.jsdelivr.net/npm/whatwg-fetch@3.0.0/dist/fetch.umd.min.js"></script>
```
Comment (jurekbarth) [inline src/layouts/includes/htmlhead.scripts.hbs]: Could we use unpkg here aswell? And one other argument for a local version: working offline, that would be a really nice to have for me

File: src/resources/ts/helper.ts [hunk 5]
```diff
@@ -14,12 +12,12 @@
...
 		biotope.configuration.set('biotope.IE', true);
-		$('html').addClass('ie');
+		document.querySelector('html').classList.add('ie');
 	} else {
 		biotope.configuration.set('biotope.IE', false);
 	}
 
 	// better touch detection
-	biotope.configuration.set('biotope.touch', (Modernizr.touch || 'ontouchstart' in window || navigator.MaxTouchPoints > 0 || navigator.msMaxTouchPoints > 0));
+	biotope.configuration.set('biotope.touch', (Modernizr.touchevents || 'ontouchstart' in window || navigator.maxTouchPoints > 0 || navigator.msMaxTouchPoints > 0));
```
Comment (jurekbarth) [inline src/resources/ts/helper.ts]: Is that the up to date version? pointer-events with fallback for Safari would be the way to go i guess.

**Pull request merged**